test/parser-specs.js

```diff
@@ -83,6 +83,27 @@ describe('Server Parser', function () {
     parsed.allowInsecure.should.eql(['feature1', 'feature2', 'feature3']);
     parsed.denyInsecure.should.eql(['nofeature1', 'nofeature2', 'nofeature3']);
   });
+  it('should parse default driver args correctly from a string', function () {
+    let fakeArgs = `{"sillyWebServerPort":1234,"host":"hey"}`;
+    let fakeDriverName = `fake`;
+    let fakeDriverArgs = `{"${fakeDriverName}": ${fakeArgs}}`;
+    let args = p.parse_args(['--driver-args', fakeDriverArgs]);
+    args.driverArgs.should.eql(JSON.parse(fakeDriverArgs));
+  });
+  it('should parse default driver args correctly from a file', function () {
+    let fakeArgs = `{"sillyWebServerPort":1234,"host":"hey"}`;
+    let fakeDriverName = `fake`;
+    let fakeDriverArgs = `{"${fakeDriverName}": ${fakeArgs}}`;
+    let args = p.parse_args(['--driver-args', 'test/fixtures/driverArgs.json']);
```

Comment:
probably want to `path.resolve(__dirname, 'fixtures', 'driverArgs.json')` so it works on windows and isn't dependent on the cwd

Reply:
(may also need to go up to the root and back down because .json fixtures might not be copied into the transpiled code)